src/eventbridge/send.ts

```diff
@@ -53,4 +54,12 @@ const sendEvents = async (testResults: TestActivity[]): Promise<SendResponse> =>
   return sendResponse;
 };
 
+const buildDebugMessage = (event: Differences | TestActivity): string =>
+  isActivity(event)
+    ? ` (testResultId: '${event.testResultId}', vin: '${event.vin}').`
+    : ` (reasonForCreation: '${event.reason}').`;
+
+// eslint-disable-next-line no-prototype-builtins
+const isActivity = (event: Differences | TestActivity): event is TestActivity => event.hasOwnProperty('vin');
```

Comment:
What is this isActivity checking? Wouldn't any test result record created be classed as an activity?

Reply:
because there could be two types of event `Differences | TestActivity`. `Differences` doesn't have `vin` property in the interface. This is effectively a way of type checking at both compile-time and run-time.

It's only used for the logger debug message above, essentially making sure the debug message logs information relevant to `Differences` when it's a `Differences` event and similarly, log relevant information for `Test Activity` when it's a `Test Activity`.

it's equivalent to doing this:
```js
const buildDebugMessage = (event: Differences | TestActivity): string =>
  event.hasOwnProperty('vin')
    ? ` (testResultId: '${(event as TestActivity).testResultId}', vin: '${(event as TestActivity).vin}').`
    : ` (reasonForCreation: '${(event as Differences).reason}').`;
```
but just cleaner because you don't have to type cast it